sqlglot/dialects/exasol.py

```diff
@@ -176,6 +178,45 @@ class Parser(parser.Parser):
                 this=self._match(TokenType.IS) and self._parse_string(),
             ),
         }
+        FUNCTION_PARSERS = {
+            **parser.Parser.FUNCTION_PARSERS,
+            # https://docs.exasol.com/db/latest/sql_references/functions/alphabeticallistfunctions/listagg.htm
+            # https://docs.exasol.com/db/latest/sql_references/functions/alphabeticallistfunctions/group_concat.htm
+            **dict.fromkeys(("GROUP_CONCAT", "LISTAGG"), lambda self: self._parse_group_concat()),
+        }
+
+        def _parse_group_concat(self) -> t.Optional[exp.Expression]:
```

Comment:
Let's avoid copy-pasting this by moving it to `dialect.py`. Then, both MySQL and Exasol can leverage the same logic.

Reply:
Alright